ctapipe/io/tableio.py

```diff
@@ -350,26 +350,106 @@ class FixedPointColumnTransform(ColumnTransform):
 
     Can be used to store values as fixed point by using an integer dtype
     and a scale that is a power of 10.
+
+    The transforms reserves 3 integers to represent -inf, nan and inf.
+    Underflowing values are converted to -inf and overflowing to inf.
+
+    For unsigned target dtype:
+    -inf: maxval - 2
+    nan: maxval - 1
+    inf: maxval
+
+    For signed target dtype:
+    -inf: minval
+    nan: minval + 1
+    inf: maxval
+
+    When reading, these special values must not be interpreted as valid integer
+    values but be transformed back into -inf, nan, inf respectively.
+
+    This is a lossy transformation.
     """
 
     def __init__(self, scale, offset, source_dtype, target_dtype):
         self.scale = scale
         self.offset = offset
         self.source_dtype = np.dtype(source_dtype)
         self.target_dtype = np.dtype(target_dtype)
+        self.unsigned = self.target_dtype.kind == "u"
+
+        iinfo = np.iinfo(self.target_dtype)
+
+        # use three highest values for nan markers for unsigned case
+        if self.unsigned:
+            self.neginf = iinfo.max - 2
+            self.nan = iinfo.max - 1
+            self.posinf = iinfo.max
+
+            # this leaves this inclusive range for the valid values
+            self.minval = 0
```

Comment:
Thats obviously the same, but I would personally use `iinfo.min` for consistency or drop a note

Reply:
Really? I find 0 much clearer than people needing to parse `iinfo.min` and think "ah right, this is unsigned, so this must be 0`....

Reply:
For me it was an overhead with all the `iinfo.min/max` above and below.
Its probably pure preference